src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  width: 100%;
+  margin-top: 20px;
+  position: relative;
+  height: 70px;
+}
+
+.img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.search {
+  font-family: avenir, sans-serif;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-size: 16px;
+  padding-left: 62px;
+  height: 70px;
+  line-height: 22px;
+}
+
+.search::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  text-shadow: 1px 1px 2px rgb(190, 186, 186);
+}
+
+.search__small {
+  font-weight: 400;
+  font-size: 14px;
+  padding-left: 33px;
+  height: 42px;
+  line-height: 20px;
+}
+
+.search__small::placeholder {
+  font-size: 14px;
+}
+
+.form__small {
+  height: 42px;
+}
+
+.img-small {
+  position: absolute;
```

Comment:
no need to duplicate the position. specify only those styles which are different